core/src/main/scala/kafka/admin/AdminClient.scala

```diff
@@ -145,19 +145,31 @@ class AdminClient(val time: Time,
 
   def describeConsumerGroup(groupId: String): Option[List[ConsumerSummary]] = {
     val group = describeGroup(groupId)
-    if (group.state == "Dead")
-      return None
+
+    group.state match {
+      case "Dead" =>
+        println(s"Consumer group `${groupId}` does not exist.")
```

Comment:
Not sure stuff in `AdminClient` should be printing to stdout. Would it make more sense to modify the return value so it can include information about dead/empty groups? For example, would it make sense to change it from an `Option[List[ConsumerSummary]]` to something like `(String, Option[List[ConsumerSummary]])`.

In fact, maybe it makes sense to just cut out the middle man here -- `describeConsumerGroup isn't really doing that much in converting the `GroupSummary` to a list of `ConsumerSummary`s. It looks like the main reason for having it factored out into `AdminClient` instead of `ConsumerGroupCommand` is for the unit test.